matlab/test/arrow/tabular/tTabularDisplay.m

```diff
@@ -124,6 +153,167 @@ function ManyRowsAndColumns(testCase, TabularType)
 
             testCase.verifyEqual(actualDisplay, expectedDisplay);
         end
+
+        function veryWideTabular(testCase, TabularType)
```

Comment:
Could you please use `UpperCamelCase` for the test names to follow the convention used in this file?

Reply:
Good catch. Thank you. I'll update the test method name to follow the convention.